doc/source/installation.rst

```diff
@@ -103,6 +103,22 @@ Ray can be built from the repository as follows.
   cd ray/python
   pip install -e . --verbose  # Add --user if you see a permission denied error.
 
+
+[Optional] Build the dashboard
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you would like to use the dashboard, you will additionally need to install
+`Node.js`_. Then you can build the dashboard as follows.
+
+.. _`Node.js`: https://nodejs.org/
+
+.. code-block:: bash
+
+  cd ray/python/ray/dashboard/client
+  npm ci
+  npm run build
+
```

Comment:
What happen after `npm run build`. Can you specify the "next step"? 
- run regular ray init with `include_webui=True`